assets/css/nav-styles.css

```diff
@@ -196,7 +196,7 @@ input:checked ~ .toggle-container {
 }
 
 .container img {
-  height: 300px;
+  height: 200px;
```

Comment:
don't set height. let the content take whatever height it requires. if it's really necessary, use `max-height` instead